### PR TITLE
 Fixed TensorFlow version check in object_detection_tutorial.ipynb

### PR DESCRIPTION
**Pull request by roffel**
The Object_detection_tutorial is compatible with TensorFlow 1.10 but the check says that version 1.4 is newer than version 1.10. This change uses the StrictVersion check from [PEP-0386 ](https://www.python.org/dev/peps/pep-0386/) to ensure good version checks in the future.

Solves #5069, #3033

I have also changed `tensorflow` to `TensorFlow` in the error message.

### CODE REVIEW

**Comment by dcyoung** [comment]
Might want to increase the minimum version from `v1.4.0` to `v1.9.0` per the requirements.

File: research/object_detection/object_detection_tutorial.ipynb [hunk 1]
```diff
@@ -53,8 +54,8 @@
         "sys.path.append(\"..\")\n",
         "from object_detection.utils import ops as utils_ops\n",
         "\n",
-        "if tf.__version__ \u003c '1.4.0':\n",
-        "  raise ImportError('Please upgrade your tensorflow installation to v1.4.* or later!')\n"
+        "if StrictVersion(tf.__version__) \u003c StrictVersion('1.4.0'):\n",
+        "  raise ImportError('Please upgrade your TensorFlow installation to v1.4.* or later!')\n"
```
Comment (pkulzc) [inline research/object_detection/object_detection_tutorial.ipynb]: v1.9

Comment (roffel) [reply research/object_detection/object_detection_tutorial.ipynb]: I'll update it in a couple of minutes!

Comment (roffel) [reply research/object_detection/object_detection_tutorial.ipynb]: @pkulzc done!

**Comment by roffel** [comment]
@pkulzc Done!

**Pull request merged**